nettoyage.py

```diff
@@ -383,11 +386,49 @@ def data_inputation(df):
 
 
 def replace_char(df):
-    """Fonction pour mettre en qualité le champ objetMarche"""
+    """Fonction pour mettre en qualité le champ objet"""
     # Remplacement brutal du caractère (?) par un espace
     df['objet'] = df['objet'].str.replace('�', 'XXXXX')
```

Comment:
```suggestion
    df['objet'] = df['objet'].str.replace('�', ' ')
```
`xxxx` ce n'est pas un espace ? ` ` la il y en à un